tests/stress/km/stress_tests_km.cpp

```diff
@@ -13,6 +13,8 @@
 #include "socket_helper.h"
 #include "socket_tests_common.h"
 
+constexpr int MAX_TAIL_CALL_PROGS = 32;
```

Comment:
As noted earlier, I think this is not needed if you can just include `bpf_helper.h`

Reply:
Fixed. Thanks